packages/dapp/hooks/useGallery.js

```diff
@@ -31,7 +28,7 @@ const useGallery = () => {
 
       const provider = new providers.JsonRpcProvider(rpcUrl);
 
-      const contract = new Contract(CONTRACT_ADDRESS, ABI, provider);
+      const Ethernauts = new Contract(CONTRACT_ADDRESS, ABI, provider);
```

Comment:
Just changed the name from `contract` to `Ethernauts` to be consistent with the backend.